can-stache-bindings.js

```diff
@@ -1202,9 +1208,29 @@ var decodeAttrName = function(name){
 var makeDataBinding = function(node, el, bindingData) {
 	// Get information about the binding.
 	var bindingInfo = getBindingInfo(node, bindingData.attributeViewModelBindings, bindingData.templateType, el.nodeName.toLowerCase(), bindingData.favorViewModel);
+
 	if(!bindingInfo) {
 		return;
 	}
+
+	//!steal-remove-start
+	if(
+		bindingInfo.child === "viewModel" &&
+		!isBindingsAttribute(bindingInfo.bindingAttributeName)
+	) {
+		var name = node.nodeName || node.name;
+		var value = node.nodeValue || node.value;
+		var filename = bindingData.scope.peek("scope.filename");
+		var lineNumber = bindingData.scope.peek('scope.lineNumber');
+
+		dev.warn(
+			(filename ? filename + ": " : "") +
```

Comment:
Sorry to nitpick, but I think these warnings look weird when there is a space before the line number. Can you update it so it looks like
`file.stache:1: foo=...`?